src/utils/composeTree.js

```diff
@@ -25,4 +25,31 @@ const composeTree = (hierarchy, properties, filterType = null) => {
   return composeTreeRecursive(hierarchy, filterType);
 };
 
+/**
+ * Reorders tree to conform to a specific ordering.
+ *
+ * `order` is a list containing the property keys that we want to reorder.
+ *
+ */
+const setHierarchyOrder = (inputHierarchy, order = null) => {
+  if (!order) {
+    return inputHierarchy;
+  }
+
+  return inputHierarchy
+    .filter((root) => root.children !== undefined)
```

Comment:
If you do `filter((root) => root.children)` does this achieve the same?

Reply:
Deleted together with the function